pkg/k8s/apis/cilium.io/v2/client/register.go

```diff
@@ -95,13 +95,18 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		return createIdentityCRD(clientset)
 	})
 
+	// Reviewers: I'm not sure if it makes sense to wrap this under the feature
+	// flag check since we have related Makefile changes.
+	// if option.Config.EnableLocalRedirectPolicy {
 	g.Go(func() error {
 		return createCLRPCRD(clientset)
 	})
+	// }
 
-	g.Go(func() error {
-		return createCCLRPCRD(clientset)
-	})
+	// CCLRP support will be added in future release.
+	// g.Go(func() error {
+	// 	return createCCLRPCRD(clientset)
+	// })
```

Comment:
Please remove this code instead.

Reply:
Ditto below for Clusterwide LRP functionality. We should not include commented out code in the codebase.